slang-kotlin/src/test/java/com/sonarsource/slang/kotlin/KotlinParserTest.java

```diff
@@ -76,6 +80,20 @@ public void testLiterals() {
       .isEquivalentTo(slangStatements("554; true; false; null; \"string\"; 'c';"));
   }
 
+  @Test
+  public void testEscapedStringLiterals() throws IOException {
+    String content = new String(Files.readAllBytes(Paths.get("src/test/resources/strings.kt")), UTF_8);
```

Comment:
I think it would be better to not have the literals in an external file. We can maybe create a variable with an explicit name to make it understandable, e.g.: `Tree escapedBackslash = kotlin("\"\\\\\"")`.